src/test/objectstore/BitAllocator_test.cc

```diff
@@ -449,7 +452,7 @@ TEST(BitAllocator, test_bmap_alloc)
   alloc->alloc_blocks(16, &start_block);
   delete alloc;
 
-  total_blocks = pow(BITMAP_SPAN_SIZE, 2) * 4;
+  total_blocks = pow(zone_size, 2) * 4;
   alloc_size = 16;
   alloc = new BitAllocator(total_blocks, zone_size, CONCURRENT, false);
   for (int64_t iter = 0; iter < 3; iter++) {
```

Comment:
> @xiexingguo is this change because you hit some issue while testing multiple zone size?

Yes, it causes many ut failures.



> Actually I wanted to keep span size and zone size independent as zone size controls maximum contiguous allocation that can be requested and span size controls height of the bitmap tree. Unless you see any logical reason both cannot coexist, I would request to keep them separate.

Well, I'll try to get ut working without this patch too.